frontend/components/Maplist.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="container">
+    <ListContainer class="listcontainer" />
+    <Map class="map" />
+  </div>
+</template>
+<script>
+import Map from "./Map.vue";
+import ListContainer from "./ListContainer.vue";
+
+export default {
+  components: {
+    Map: Map,
+    ListContainer: ListContainer,
+  },
```

Comment:
If field name is the same as a value, you can do it like this:

```js
  components: {
    Map,
    ListContainer,
  },
  ```

Reply:
Fixed.